DTIPrep/DTIPrepRegister.pl

```diff
@@ -43,14 +43,14 @@ =head1 DESCRIPTION
 The following output files will be inserted:
   - QCed MINC file produced by DTIPrep pre-processing step (i.e. DWI
      dataset without the bad directions detected by DTIPrep)
-  - QCReport produced by DTPrep
-  - XMLQCResult produced by DTIPrep
-  - RGB map produced by either DTIPrep or mincdiffusion post-processing
-  - MD map produced by either DTIPrep or mincdiffusion post-processing
-  - FA map produced by either DTIPrep or mincdiffusion post-processing
-  - baseline image produced by DTIPrep or mincdiffusion post-processing
-  - DTI mask produced by mincdiffusion post-processing (only if
-     mincdiffusion was used to post-process the data)
+  - Text QC report produced by DTPrep
```

Comment:
here too, if you made DTIPrep as a C<>, it should be done across here and other scripts as well?